fiwareskuld/expired_users.py

```diff
@@ -150,9 +151,9 @@ def _get_red_yellow(self, users):
                 # a week or less.
                 yellowList.append(user)
 
-        logger.info("Number of expired Trial Users found: %d",
+        logger.info("Number of expired Users found: %d",
```

Comment:
No, we should differentiate between Trial and Community users, therefore, the logger info should show both detailed data.